app/views/documents/history.html.erb

```diff
@@ -15,61 +15,15 @@
           <% displayed_edition_number = entry.edition.number %>
         <% end %>
 
-        <% if entry.imported_from_whitehall? %>
+        <% if entry.whitehall_migration? && entry.details.imported_from_whitehall? %>
           <%= render "govuk_publishing_components/components/notice", {
-            title: "#{t "documents.history.entry_types.#{entry.entry_type}"}",
+            title: "#{t "documents.history.entry_types.whitehall_migration.#{entry.details.entry_type}"}",
             description_text: format_date(entry.created_at)
           } %>
```

Comment:
I think it'd make sense that this entry was in the whitehall_migration partial, that way all whitehall imported entries are in the same file.